has_many_blogs.sql

```diff
@@ -0,0 +1,37 @@
+DROP DATABASE IF EXISTS has_many_blogs;
```

Comment:
These three lines are not needed since docker will create the database and user for you.